pyproject.toml

```diff
@@ -153,3 +157,13 @@ select = [
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.envs.docs]
```

Comment:
Can you add some scripts too:

```
https://github.com/mrc-ide/hipercow-py/blob/main/pyproject.toml#L196-L198
```

then we can do `hatch run build` or `hatch run serve`

Reply:
I've been trying various things but couldn't get this to work - I get a 'command not found' error locally. `mkdocs serve` and `build` work fine for me though.